website/docs/r/vapp_network.html.markdown

```diff
@@ -50,7 +50,11 @@ The following arguments are supported:
 * `name` - (Required) A unique name for the network.
 * `description` - (Optional; *v2.7+*, *vCD 9.5+*) Description of vApp network
 * `vapp_name` - (Required) The vApp this network belongs to.
-* `netmask` - (Optional) The netmask for the new network. Default is `255.255.255.0`.
+* `netmask` - (Deprecated) Use `prefix_length` instead. The netmask for the new network.
```

Comment:
My main concern is that we are removing the default value, which means users will have to amend existing configurations. This in turn means this is a breaking change, which would affect all configurations which have vApp network out there?

CC: @Didainius @dataclouder @adambarreiro